src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlDependency.cs

```diff
@@ -6,7 +6,16 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Globalization;
+#if NETFRAMEWORK
```

Comment:
Imo, it's a little neater to have one #if for netfx, rather than trying to maintain alphabetical order.